buddyboss/buddyboss-platform
# PROD-765 fix/2780 > Profile date field in repeater > empty value auto converts to date January 1970

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2780 .

### How to test the changes in this Pull Request:

1. Add field set with date field under it.
2. In edit profile set the date to blank.
3. You will see that it will auto convert to date January 1970.

### Proof Screenshots or Video

https://www.loom.com/share/87d246da3e21496488767eb46f041463

### Other information:


**Support ticket links**
https://secure.helpscout.net/conversation/1598403327/158236?folderId=3709082
